webapp/channels/src/components/sidebar/sidebar_category/sidebar_category_menu/mark_as_read_menu_item.tsx

```diff
@@ -11,16 +11,17 @@ import {openModal} from 'actions/views/modals';
 import {ModalIdentifiers} from 'utils/constants';
 import MarkAsReadConfirmModal from './mark_as_read_confirm_modal';
 
-type Props = ({
+type Props = {
     id: string;
     handleViewCategory: () => void;
     numChannels: number;
-})
+}
```

Comment:
Same about the Prop types.

Reply:
I'll change that, but weren't you the one that just added this component recently? 😛

I'm actually surprised that ESLint didn't remove this since it usually enforces if brackets can show up like this